src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-manual-install-infrastructure-agent-linux.mdx

```diff
@@ -56,7 +56,6 @@ Before installation, check the [compatibility and requirements](/docs/infrastruc
       </td>
     </tr>
   </tbody>
-</table>
```

Comment:
looks like this may be causing the build failure

Reply:
good catch!